### DISCUSSION TITLE
French model based on large-v2

### DISCUSSION THREAD

**Discussion by cayerpm** [openai/whisper#1325] [Q&A] [answered]
Is there a way to get a French only model based on large-v2 ? 

I need to transcribe files in French an English only. English is not really an issue with other models, but French seems to work a lot better in the large-v2 model. My issue is that with my current rig, I do not have enough VRAM to use the large-v2 model.

Upgrading the GPU might be an option, but I wanted to see if there were other options.

Thanks

PM

**Comment by phineas-pta** [comment, marked as answer]
there is no french only models, only models fine-tuned on french

if you want to use large model on but your rig doesn't have enough VRAM, you can try:

- google colab
- faster-whisper https://github.com/guillaumekln/faster-whisper
